test/test_acousticbrainz.py

```diff
@@ -101,6 +101,14 @@ def test_realistic(self):
         }
         self.assertEqual(mapping, expected)
 
+    def test_invalid_characters(self):
+        ab = AcousticPlugin()
+        data = {'key 1': "['?\x32\x96 233474']", 'key 2': '\xb9\x42'}
+        scheme = {'key 1': 'attribute 1', 'key 2': 'attribute 2'}
+        mapping = set(ab._map_data_to_scheme(data, scheme))
```

Comment:
I'm a little confused by this test… it doesn't seem like `_map_data_to_scheme` involves any JSON reading/writing, right?